src/main/java/worker/Main.java

```diff
@@ -64,12 +73,14 @@ public static void startWorker(int port) {
 
     ActorSystem system = ActorSystem.create("WorkerSystem", conf);
 
-    Set<ActorSelection> initialContacts = new HashSet<ActorSelection>();
+    Set<ActorPath> initialContacts = new HashSet<>();
     for (String contactAddress : conf.getStringList("contact-points")) {
-      initialContacts.add(system.actorSelection(contactAddress + "/user/receptionist"));
+      initialContacts.add(ActorPaths.fromString(contactAddress + "/system/receptionist"));
     }
```

Comment:
My thinking was that we should anyway put it under a separate config, since this is intended as a "copy-paste" example. WDYT?